src/client/Microsoft.Identity.Client/Http/HttpManager.cs

```diff
@@ -194,9 +194,12 @@ private HttpClient GetHttpClient(X509Certificate2 x509Certificate2, Func<HttpReq
             return _httpClientFactory.GetHttpClient();
         }
 
+        private static readonly Version s_http2Version = new Version(2, 0);
+
         private static HttpRequestMessage CreateRequestMessage(Uri endpoint, IDictionary<string, string> headers)
         {
             HttpRequestMessage requestMessage = new HttpRequestMessage { RequestUri = endpoint };
+            requestMessage.Version = s_http2Version;
```

Comment:
will we automatically drop to 1.1 where not  supported?

Reply:
Cool, I didn't know about this.